get-started/what-is-shiny.qmd

```diff
@@ -0,0 +1,160 @@
+---
+title: What is Shiny?
+---
+
+```{=html}
+<style>
+  .btn-links {
+    display: flex;
+    gap: 1.5rem;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .btn-link-large {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 150px;
+    height: 150px;
+    font-size: 1.5rem;
+    border: 1px solid transparent;
+    border-radius: 10px;
+    cursor: pointer;
+    padding: 1.25rem;
+    text-decoration: none;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  }
+
+  .btn-link-large i {
+    margin-top: 10px;
+    font-size: 2rem;
+  }
+
+  .btn-link-large:hover {
+    text-decoration: none;
+    border: 1px solid var(--bs-primary);
+  }
+</style>
+```
+
+Whether you’re a data scientist, analyst, or developer, Shiny makes it easy to [create](create-run.qmd) rich, [interactive experiences](/docs/ui-overview.qmd) in pure Python [without needing](/docs/reactive-foundations.qmd) to learn JavaScript or front-end frameworks.
+
+## Batteries included {#batteries-included}
+
+Shiny for Python comes fully equipped with everything you need to build a dashboard right out of the box, including a rich set of [input](/components/#inputs) and [output](/components/#outputs) [components](/components/) so you can quickly build interactive applications.
```

Comment:
Another link comment: visually there's no separation between `output` and `components` so they look like one continuous link when you're viewing the page. In general, it's best to avoid directly adjacent links unless there's a word or punctuation between the links. In this case, I don't think we gain anything from linking "components", so I'd recommend dropping that link.

```suggestion
Shiny for Python comes fully equipped with everything you need to build a dashboard right out of the box, including a rich set of [input](/components/#inputs) and [output](/components/#outputs) components so you can quickly build interactive applications.
```